### PR TITLE
Improve german translations

### PR DESCRIPTION
**Pull request by raphael-s**
IMO some of the german translations needed some improvement, so I tried my best to update them.

I don't know if that is useful by any means 😄

### CODE REVIEW

**Comment by raphael-s** [comment]
@mbaechtold Added an entry to the changelog 👍

File: ftw/collectionblock/profiles/default/types/ftw.collectionblock.CollectionBlock.xml [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
     <!-- Basic metadata -->
     <property name="title" i18n:translate="">Collection block</property>
-    <property name="description" i18n:translate="">Build complex queries with easy inside a simplelayout block.</property>
+    <property name="description" i18n:translate="">Easily build complex queries inside a simplelayout block.</property>
```
Comment (mbaechtold) [inline ftw/collectionblock/profiles/default/types/ftw.collectionblock.CollectionBlock.xml]: This requires to be changed through an upgrade step for existing installations.

**Comment by raphael-s** [comment]
@mbaechtold I added an upgrade step to update the new description.

**Comment by raphael-s** [comment]
@mbaechtold I rebased this branch onto master after mergin the other PR. I also did a rerun of i18n-build which updated some line numbers in the translation files.

**Pull request merged**